DBALEventStoreException during any database operation

Hey, I'm new into CQRS and Broadway, but I'm issuing with error I cannot solve myself. I've added some fixtures to this app, while loading data to database I'm getting an error:

```
[Broadway\EventStore\Dbal\DBALEventStoreException]  
                                                      
In AbstractMySQLDriver.php line 42:
                                                                               
  An exception occurred while executing 'INSERT INTO event (uuid, playhead, m  
  etadata, payload, recorded_on, type) VALUES (?, ?, ?, ?, ?, ?)' with params  
   ["\x4b\x80\xf6\x6c\xa5\x46\x47\x73\x92\x55\xb5\xcb\xe1\x0f\xa2\x39", 0, "{  
  \"class\":\"Broadway\\\\Domain\\\\Metadata\",\"payload\":[]}", "{\"class\":  
  \"App\\\\Domain\\\\User\\\\Event\\\\UserWasCreated\",\"payload\":{\"uuid\":  
  \"4b80f66c-a546-4773-9255-b5cbe10fa239\",\"credentials\":{\"email\":\"test1  
  @test.com\",\"password\":\"$2y$12$ESU377hWQsynNogPGz\\\/26.\\\/9RaBcnhkeNEn  
  Uz7ZxItmlNe7Rn07yu\"},\"created_at\":\"2020-06-06T09:48:30.595163+00:00\"}}  
  ", "2020-06-06T09:48:30.595621+00:00", "App.Domain.User.Event.UserWasCreate  
  d"]:                                                                         
                                                                               
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'cloudsso.event'   
  doesn't exist
```

I've done new migration, I've run it successfully, but it didn't created events table required by DBALEventStore. 

Thanks in advance for your support!

Hmm, I've had lots of problems during moving my project from simple Symfony architecture into CQRS & ES one, so I decided to clone this repo and use it as a boilerplate. Anyway, now I'm getting an error like this:

```
docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml stop
Stopping cloud-sso_elasticsearch_1 ... done
Stopping cloud-sso_mysql_1         ... done
Stopping cloud-sso_rmq_1           ... done
docker-compose rm -v -f
Going to remove cloud-sso_elasticsearch_1, cloud-sso_mysql_1, cloud-sso_rmq_1
Removing cloud-sso_elasticsearch_1 ... done
Removing cloud-sso_mysql_1         ... done
Removing cloud-sso_rmq_1           ... done
docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml build
mysql uses an image, skipping
elasticsearch uses an image, skipping
rmq uses an image, skipping
php uses an image, skipping
nginx uses an image, skipping
workers uses an image, skipping
kibana uses an image, skipping
docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml run --rm php sh -lc 'xoff;COMPOSER_MEMORY_LIMIT=-1 composer install'
Creating cloud-sso_elasticsearch_1 ... 
Creating cloud-sso_rmq_1           ... 
Creating cloud-sso_mysql_1         ... 
Creating cloud-sso_mysql_1         ... error

Creating cloud-sso_elasticsearch_1 ... doneCreating cloud-sso_rmq_1           ... doneted

ERROR: for mysql  Cannot start service mysql: driver failed programming external connectivity on endpoint cloud-sso_mysql_1 (cc8f6155dd0f88a8439dd7039d7f3c86fdd8d202949226a4cde3291b0a9c6f3d): Bind for 0.0.0.0:3306 failed: port is already allocated
ERROR: Encountered errors while bringing up the project.
make: *** [makefile:31: build] Error 1
```

I know it's mysql-related, but I have no idea what's going on.

Whole console output:
```sh
archi_tektur@Oskar-PC:~/Projects/Cloud/cloud-core$ make start
docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml stop
Stopping cloud-core_nginx_1         ... done
Stopping cloud-core_workers_1       ... done
Stopping cloud-core_php_1           ... done
Stopping cloud-core_kibana_1        ... done
Stopping cloud-core_rmq_1           ... done
Stopping cloud-core_elasticsearch_1 ... done
docker-compose rm -v -f
Going to remove cloud-core_nginx_1, cloud-core_workers_1, cloud-core_php_1, cloud-core_mysql_1, cloud-core_rmq_1, cloud-core_elasticsearch_1
Removing cloud-core_nginx_1         ... done
Removing cloud-core_workers_1       ... done
Removing cloud-core_php_1           ... done
Removing cloud-core_mysql_1         ... done
Removing cloud-core_rmq_1           ... done
Removing cloud-core_elasticsearch_1 ... done
docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml build
rmq uses an image, skipping
mysql uses an image, skipping
kibana uses an image, skipping
elasticsearch uses an image, skipping
workers uses an image, skipping
php uses an image, skipping
nginx uses an image, skipping
docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml run --rm php sh -lc 'xoff;COMPOSER_MEMORY_LIMIT=-1 composer install'
Creating cloud-core_elasticsearch_1 ... done
Creating cloud-core_rmq_1           ... done
Creating cloud-core_mysql_1         ... done
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
81 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Executing script cache:clear [OK]
Executing script assets:install public [OK]

docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml up -d
Starting cloud-core_mysql_1 ... 
cloud-core_rmq_1 is up-to-date
Starting cloud-core_mysql_1  ... done
Starting cloud-core_kibana_1 ... done
Creating cloud-core_workers_1 ... done
Creating cloud-core_php_1     ... done
Creating cloud-core_nginx_1   ... done
docker-compose -f docker-compose.yml -f etc/dev/docker-compose.yml exec -T php sh -lc './bin/console d:d:d --force --if-exists'

In AbstractMySQLDriver.php line 93:
                                                                               
  An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddr  
  esses: getaddrinfo failed: Name does not resolve                             
                                                                               

In PDOConnection.php line 31:
                                                                               
  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name d  
  oes not resolve                                                              
                                                                               

In PDOConnection.php line 27:
                                                                               
  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name d  
  oes not resolve                                                              
                                                                               

In PDOConnection.php line 27:
                                                                               
  PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name does  
   not resolve                                                                 
                                                                               

doctrine:database:drop [-s|--shard SHARD] [-c|--connection [CONNECTION]] [--if-exists] [-f|--force] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>

make: *** [makefile:81: db] Error 1
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -64,3 +64,11 @@ services:
     environment:
       - "discovery.type=single-node"
       - "cluster.name=sf-events"
+
+  start_dependencies:
+    image: dadarek/wait-for-dependencies
+    depends_on:
+      - mysql
+      - rmq
+      - elasticsearch
+    command: mysql:3306 elasticsearch:9200 rmq:5672

--- a/makefile
+++ b/makefile
@@ -11,7 +11,11 @@ endif
 export compose env docker-os
 
 .PHONY: start
-start: erase build up db ## clean current environment, recreate dependencies and spin up again
+start: erase build start-deps up db ## clean current environment, recreate dependencies and spin up again
+
+.PHONY: start-deps
+start-deps:  ## Start all dependencies and wait for it
+		$(compose) run --rm start_dependencies
 
 .PHONY: stop
 stop: ## stop environment

